test/Spec/State/Computed.spec.luau

```diff
@@ -71,7 +71,7 @@ return function()
 		
 		local scope = {}
 		local dependency = Value(scope, 5)
-		local computed = Computed(scope, function(use)
+		local computed = Computed(scope, function(use, scope)
 			assert(use(dependency) ~= 13, "This is an intentional error from a unit test")
```

Comment:
Scope shouldn't be provided as it is never used
```suggestion
		local computed = Computed(scope, function(use)
```

Reply:
reverted